cmake/extensions.cmake

```diff
@@ -38,7 +42,7 @@ function(target_cddl_source target cddl_file)
     message(FATAL_ERROR "CDDL input file ${cddl_file} does not exist.")
   endif()
 
-  cmake_parse_arguments(CDDL "DECODE;ENCODE;VERBOSE;CANONICAL" "" "ENTRY_TYPES" ${ARGN})
+  cmake_parse_arguments(CDDL "DECODE;ENCODE;VERBOSE;CANONICAL" "TYPE_FILE_NAME" "ENTRY_TYPES" ${ARGN})
```

Comment:
Maybe using the traditional "required" and "optional" lists would make sense now that both types are in use? (At least that what it looks like to me)

Reply:
Do you mean a "required" and "optional" list in the documentation? Probably a good idea.

Reply:
done